contracts/veLON.sol

```diff
@@ -463,14 +492,14 @@ contract veLON is IveLON, ERC721, Ownable, ReentrancyGuard {
         // storedPoolLastPoint is used for extrapolation to calculate block number
         // (approximately, for *At methods) and save them
         // as we cannot figure that out exactly from inside the contract
-        Point memory storedPoolLastPoint = poolLastPoint;
+        Point memory storedPoolLastPoint = Point({ vBalance: 0, decliningRate: 0, ts: poolLastPoint.ts, blk: poolLastPoint.blk });
 
-        // blockRate = dBlock/dTime
+        // blockRate = dTime/dBlock
         // If last point is already recorded in this block, blockRate=0
         // But that's ok because we know the block in such case
         uint256 blockRate = 0;
```

Comment:
Rename as `avgBlkTime`.

Reply:
rename in https://github.com/consenlabs/tokenlon-contracts/pull/81/commits/ef590c464974ef594ff79e9bae2d6d986f5b114d